apps/webpack.js

```diff
@@ -30,6 +30,7 @@ var baseConfig = {
       '@cdo/apps': path.resolve(__dirname, 'src'),
       '@cdo/static': path.resolve(__dirname, 'static'),
       repl: path.resolve(__dirname, 'src/noop'),
+      http: 'stream-http',
```

Comment:
The weirdness around the Webpack 1 vs. Webpack 2 node polyfills?  Or are you thinking of something different?